pkg/reconciler/tenancy/clusterworkspace/clusterworkspace_reconcile_phase.go

```diff
@@ -62,22 +59,6 @@ func (r *phaseReconciler) reconcile(ctx context.Context, workspace *tenancyv1alp
 			return reconcileStatusContinue, nil
 		}
 
-		bindings, err := r.getAPIBindings(logicalcluster.From(workspace).Join(workspace.Name))
-		if err != nil {
-			return reconcileStatusContinue, err
-		}
-		var unbound []string
-		for _, binding := range bindings {
-			if !conditions.IsTrue(binding, apisv1alpha1.InitialBindingCompleted) {
-				unbound = append(unbound, fmt.Sprintf("%s:%s", binding.Name, conditions.GetReason(binding, apisv1alpha1.InitialBindingCompleted)))
-			}
-		}
-		if len(unbound) > 0 {
-			sort.Strings(unbound)
-			conditions.MarkFalse(workspace, tenancyv1alpha1.WorkspaceInitialized, tenancyv1alpha1.WorkspaceInitializedAPIBindingNotBound, conditionsv1alpha1.ConditionSeverityInfo, "APIBindings not bound: %s", strings.Join(unbound, ", "))
-			return reconcileStatusContinue, nil
-		}
-
```

Comment:
where is this part gone?

Reply:
It's handled by the new controller

Reply:
In that, if the apibindings initializer is still present, the workspace remains initialized=false